src/pool/pool_jemalloc.c

```diff
@@ -305,6 +317,8 @@ static void *je_calloc(void *pool, size_t num, size_t size) {
         return NULL;
     }
 
+    utils_annotate_memory_defined(ptr, num * size);
```

Comment:
We mark the memory in `je_calloc` and `je_realloc` as defined since its zero-initialized or copied, so why is the memory from `je_malloc` not marked as undefined?